### ISSUE TITLE
--version not working

### ISSUE THREAD

**Issue by pzuliani** [dreal/dreal3#117]
With the latest commit
```
$ ./dReal --version
# Error: can't open file (triggered at /usr/local/src/dreal3/src/opensmt/smtsolvers/SMTConfig.h, 51)
```

**Comment by soonhokong** [comment]
@pzuliani, I just fix this in 9196b87.

**Comment by soonhokong** [comment]
FYI, the filename has to come at the end of command-line arguments. This is due to the change of the parsing library (GFLAGS => EzOptionParser). If this is too annoying, I'll think about improving it..
